polaris-icons/package.json

```diff
@@ -19,6 +19,14 @@
   "files": [
     "dist/*"
   ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./metadata": "./dist/metadata.json"
```

Comment:
Exposing a json file doesn't feel that great. You can load it with `require()` but you can't load it with `import` statement (unless you muck about with running node with CLI options but that's not general purpose yet). I think it might be better to expose the metadata as a js file if possible.


Tangentally related - it looks like this file is generated with a script. It might be nicer to use rollup's virtual file stuff to create this file, in [the same way that it generates the index file](https://github.com/Shopify/polaris/blob/main/polaris-icons/rollup.config.js#L258-L260) - that way we get to keep the build as " run rollup" without needing extra steps